Sprint-2/implement/contains.js

```diff
@@ -1,3 +1,8 @@
-function contains() {}
+function contains(object, property) {
+  for (const key of Object.keys(object)) {
```

Comment:
This is good and works. Can you please see if there is another way of using inbuilt object methods to check object has property as key besides extracting the properties of object and checking for property key

Please refer: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object